src/pages/home/report/ReportActionsList.js

```diff
@@ -115,8 +115,12 @@ const ReportActionsList = (props) => {
             // When the new indicator should not be displayed we explicitly set it to null
             const shouldDisplayNewMarker = reportAction.reportActionID === newMarkerReportActionID;
             const shouldDisplayParentAction = reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.CREATED && ReportUtils.isThread(report);
+            // check if the second message of the thread is unread, then hide the divider line.
```

Comment:
I feel this comment is off, it then explain why we need to hide the divider line and on what condition we need to hide the divider line.

Reply:
Yeah, since the variable name itself was quite clear about its functioning. We can remove that. I'll update it. Thank you @mollfpr